app/assets/javascripts/govuk-admin-template/govuk-admin.js

```diff
@@ -85,16 +85,23 @@
 
   // Google Analytics event tracking
   // Label and value are optional
-  GOVUKAdmin.trackEvent = function(action, label, value) {
-
+  GOVUKAdmin.trackEvent = function(action, label, value, category) {
```

Comment:
I've added a separate card to solve this https://trello.com/c/BW5fA4ar/141-normalizes-ga-tracking-for-admin-gem